docs/prod-install.md

```diff
@@ -43,21 +32,29 @@ tar -xf zulip-server-latest.tar.gz
 If you'd like to verify the download, we
 [publish the sha256sums of our release tarballs](https://www.zulip.org/dist/releases/SHA256SUMS.txt).
 
-Then, run the Zulip install script:
+## Step 2: Install Zulip
+
+Most users will want Zulip to automatically obtain an SSL certificate
```

Comment:
What is the statistics of the userbase for this means of installation (>=a supermajority?)? Should the user have already an SSL cert ready, how to tell that both the `--email` and `--hostname` are mandatory config? Step 3 is now just for configuring outgoing email.

Reply:
I'm not worried about the statistics -- our docs on Certbot lay out the exceptions to this recommendation, so anyone can judge for themselves whether they actually fall into this class.

We'll want to make `--email` and `--hostname` mandatory, but we haven't yet. When we do, the install script should give a nice error if you omit them, and should have a nice usage message.